build.py

```diff
@@ -362,14 +362,17 @@ def reformat_for_drupal(info):
 
```

Comment:
The build.py script is going out of use and I never touched it, so I can't comment on the updates and kinda think we don't need to update it